src/app/components/planners/PlannerList.tsx

```diff
@@ -13,25 +13,25 @@ export default function PlannerList() {
 
   if (!planners || planners.length == 0) return <HelpfulTips />;
 
+  // Why using map? Why not just render the Planner component directly?
```

Comment:
Because we need to include the providers with their props. We can move these providers into Planner but then we'd need to give it an `id` prop. This might be better though.

Reply:
I'm gonna wait to see if this is something we can do away with Zustand, not sure its worth the refactor if its going to be refactored again soon.

I definitely see the need for Zustand while working through this component, I had to work through a lot of wrapping divs that were only there for providers and other non-ui elements, ideally we would simplify this at some point if possible.